etc/bash_completion.d/rudder.sh

```diff
@@ -1,8 +1,26 @@
+_global() {
+  if [ $COMP_CWORD -ge 1 ]; then
+    case ${COMP_WORDS[1]} in
+      package)
+        PKG_FILE="rudder-pkg.sh"
```

Comment:
I think you need an absolute path here since you don't know the current directory